scripts/sources.conf

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Do not edit this file, create sources.local.conf for overriding.
+#
+
+# Some random port because 8000 is typically already used.
+export PORT=8131
```

Comment:
can we also set `METRICS_PORT`? They have 9000 as well as we do and it could collide, right?

Reply:
Good point, added.